.werft/jobs/build/prepare.ts

```diff
@@ -24,7 +23,7 @@ export async function prepare(werft: Werft, config: JobConfig) {
         configureStaticClustersAccess()
         werft.done(prepareSlices.CONFIGURE_CORE_DEV)
 
-        issueCertificate(werft, config)
+        await issueCertificate(werft, config)
```

Comment:
This is just creating the Certificate CRD resources that  CertManager uses. I don't believe we want to do so as a "fire-and-forget" promise

Reply:
agreed - assuming we only create the CR and don't actually wait for the cert to be issued.